packages/website/cypress/step-definitions/base.ts

```diff
@@ -32,6 +28,14 @@ When('User types {string} in the input with placeholder {string}', (text: string
   cy.get(`input[placeholder="${placeholder}"]`).type(text);
 });
 
+When('User types {string} into the {int}st/nd/rd/th input with id {string}', (text: string, idx:number, id: string) =>{
+  cy.get(`[data-testid="${id}"]`).eq(idx - 1).clear().type(`${text}`);
```

Comment:
Is it possible for this step to add the `input` type? This is just to be consistent with the text that says `"... into the input ..."`
```suggestion
  cy.get(`input[data-testid="${id}"]`).eq(idx - 1).clear().type(`${text}`);
```

Reply:
Added `input` as your suggestion.

![Screenshot 2025-02-24 at 10 51 37](https://github.com/user-attachments/assets/4c776343-f5cb-4c24-9211-c8aa2d5a6ac8)